Resources/Prototypes/Nyanotrasen/Reagents/Consumable/Drink/alcohol.yml

```diff
@@ -37,3 +37,153 @@
       - !type:AdjustReagent
         reagent: Ethanol
         amount: 0.15
+
+- type: reagent
+  id: OrangeCreamice
+  name: reagent-name-orange-creamcicle
+  parent: BaseAlcohol
+  desc: reagent-desc-orange-creamcicle
+  physicalDesc: reagent-physical-desc-creamy
+  flavor: orangecreamcicle
+  color: "#efaa16"
+  metamorphicSprite:
+    sprite: Nyanotrasen/Objects/Consumable/Drinks/orangecreamcicle.rsi
+    state: icon
+  metabolisms:
+    Drink:
+      effects:
+      - !type:SatiateThirst
+        factor: 2
+      - !type:AdjustReagent
+        reagent: Ethanol
+        amount: 0.2
```

Comment:
It's 1 part vodka to 3 parts non-alcoholic stuff so this should be 0.05.

Reply:
Uh I guess upstream doesn't do that since Bloody Mary is 0.3 ethanol even tho it's 2 part vodka (which is 0.2 ethanol...) and 4 part non-alcoholics
Wilco.

Reply:
upstream bug then